From 85661a8c9f0774b12ad04c8eac490a668cb030d1 Mon Sep 17 00:00:00 2001
From: Albin B <albin.bala.krishnan@intel.com>
Date: Tue, 8 Dec 2015 18:37:53 +0530
Subject: [PATCH 7/7] [EM/USBC_PD]: Sink offers one pdo with max current for
 same voltage

According to the PD2.1 spec, sink shall offer only the minimal pdo
for the same power level. Hence use only one pdo with the maximum
supported current for the same voltage level.

Change-Id: I12e83c9aa16f7a077f9df3ee5f506564599c5136
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-11632
Signed-off-by: Albin B <albin.bala.krishnan@intel.com>
Reviewed-on: https://android.intel.com:443/445693
---
 drivers/usb/typec/pd/devpolicy_mgr.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/drivers/usb/typec/pd/devpolicy_mgr.c b/drivers/usb/typec/pd/devpolicy_mgr.c
index 9f547d2..9a65946 100644
--- a/drivers/usb/typec/pd/devpolicy_mgr.c
+++ b/drivers/usb/typec/pd/devpolicy_mgr.c
@@ -43,11 +43,6 @@ static struct power_cap spcaps[] = {
 	},
 	{
 		.mv = VIN_12V,
-		.ma = ICHRG_1A,
-		.psy_type = DPM_PSY_TYPE_FIXED,
-	},
-	{
-		.mv = VIN_12V,
 		.ma = ICHRG_3A,
 		.psy_type = DPM_PSY_TYPE_FIXED,
 	},
-- 
1.9.1

